packages/vue/src/components/authenticator.vue

```diff
@@ -326,6 +326,11 @@ const confirmResetPasswordComponent = ref(null);
 const verifyUserComponent = ref(null);
 const confirmVerifyUserComponent = ref(null);
 
+// computed
+
+const signInLabel = computed(() => I18n.get(CREATE_ACCOUNT_LABEL));
```

Comment:
What does `computed` do here, or how will it know if `I18n`'s value has changed?

`I18n.get` is a lookup, so is there a benefit from having it assigned to `signInLabel` vs. used directly in the view?

Reply:
The computed creates a reactive variable around the `I18n.get(CREATE_ACCOUNT_LABEL)` That way if they dynamically change the language, the computed property will get re-rendered with the correct value. 

I think this could work directly in the view, it probably will work with Vue's reactivity system, we would have to test it to see if dynamically changing the language updates correctly.  

This will work fine though

Reply:
I guess what I'm wondering is _how will Vue know when these values change_?

My understanding is that anything within `computed` relies on values defined within Vue's components or composables. 

Like, suppose this renders on the page and then you did a `setTimeout(..., 5000)` that did `I18n.put({ en: { CREATE_ACCOUNT_LABEL: 'updated' } })` (not exactly, but you get the idea) – I don't know how Vue would know to update this.

Reply:
Anytime any values in the computed property change, it gets recalculated. So if the `CREATE_ACCOUNT_LABEL` changes it gets updated. If the `I18n.get()` returns a different value, then it recalculates. It's also cached too.